.github/workflows/ci.yml

```diff
@@ -10,13 +10,12 @@ jobs:
 
     steps:
     - uses: actions/checkout@v3
-    - name: Set GCC & G++ 10 compiler (on Linux)
+    - name: Set GCC & G++ 11 compiler (on Linux)
       if: runner.os == 'Linux'
-      run: echo "CC=gcc-10" >> $GITHUB_ENV && echo "CXX=g++-10" >> $GITHUB_ENV
+      run: echo "CC=gcc-11" >> $GITHUB_ENV && echo "CXX=g++-11" >> $GITHUB_ENV
```

Comment:
unrelated, but could this simply be an `env:` entry? Or is there something special about the use of $GITHUB_ENV here?

Reply:
Looking at the git blame, I modified it in PR https://github.com/AOMediaCodec/libavif/pull/595 about one and a half years ago. I think this has to do with this issue:

- [GitHub Actions: Deprecating set-env and add-path commands](https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/)

Or it has to do with retention between steps.

But I don't remember exactly anymore. We could try if it works with a regular `env:`, if you want.

Reply:
For this change we can leave it as is. I was just curious when comparing other workflow files.